src/components/DigitalAssetsGrid/AddAsset/AddAsset.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import React from 'react'
+
+import {
+  JIcon,
+  JLink,
+} from 'components/base'
+
+export function AddAsset() {
+  return (
+    <JLink
+      href='/assets'
+      className='add-asset'
+    >
+      <div className='icon'>
```

Comment:
JLink is `<a` that semantic inline element, so I think it is better to use `span` here

Reply:
+1
`span` is good choice, but, meanwhile, in HTML5 specs content type for anchors is don't matter.
https://www.w3.org/TR/2011/WD-html-markup-20110405/a.html#a-content-model